src/client/pages/Registration.tsx

```diff
@@ -42,40 +92,45 @@ export const Registration = ({ returnUrl = '', email = '' }: Props) => {
           },
         ]}
       />
-      <Main>
+
+      <Main
+        errorOverride={
+          recaptchaCheckSuccessful ? undefined : CaptchaErrors.GENERIC
+        }
+      >
         <PageBox>
           <PageBody>
             <form
-              css={form}
               method="post"
-              action={`${Routes.REGISTRATION}${returnUrlQuery}`}
+              action={`${Routes.REGISTRATION}?${registrationUrlQueryParamString}`}
+              ref={registerFormRef}
+              onSubmit={handleSubmit}
             >
               <CsrfFormField />
+              <RecaptchaElement
+                ref={recaptchaElementRef}
+                id="register-recaptcha"
+              />
               <TextInput
                 css={textInput}
                 label="Email"
                 name="email"
                 type="email"
                 defaultValue={email}
               />
-              <TextInput
-                css={textInput}
-                label="Password"
-                name="password"
-                type="password"
-              />
+              <div css={termsSpacing}>
+                <Terms />
+              </div>
               <Button css={button} type="submit" data-cy="register-button">
                 Register
               </Button>
             </form>
             <Divider
-              size="full"
+              size="fit"
```

Comment:
Fit the divider so that the lines don't overlap the width of the form

Reply:
Kept as `full` and `loose` for now, because these options need to be added in the Source Kitchen.